### DISCUSSION TITLE
Unhandled Runtime Error TypeError: Cannot read properties of undefined (reading 'connector')

### DISCUSSION THREAD

**Discussion by saidul112** [WalletConnect/walletconnect-monorepo#3945] [Developer Support]
Hey, I am using https://docs.walletconnect.com/web3modal/react/about?platform=wagmi into my React project. I have integrated in one file. But during run time i get this error "https://docs.walletconnect.com/web3modal/react/about?platform=wagmi" 
<img width="959" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/87698836/d79b2a7c-b416-4fe6-966c-94d7f16f8198">
<img width="959" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/87698836/6ba13f7f-eaba-4af5-bee1-c7b7c599cdee">

**Comment by crypblizz8** [comment]
Closing as it is a repeat of #3946
